### DISCUSSION TITLE
CGO-free sqlite

### DISCUSSION THREAD

**Discussion by MarketDataApp** [maragudk/goqite#53] [Ideas]
I came across your library looking for a task-queue for Go that uses sqlite. We're looking to build a data terminal that can be widely released, so for our use case it would be interesting if this could work with https://pkg.go.dev/modernc.org/sqlite as this does not require CGO for cross-compilation. It would make the build process & release process easier.

I like your focus on sqlite and we may end up using your package.

**Comment by markuswustenberg** [comment]
As it says in the top of the readme, you can use whatever driver you want. :)

**Reply by MarketDataApp** [reply]
Sorry, missed that in the readme. That's great to hear. I'm going to start testing this. 

Thanks for the quick reply and for open sourcing this.

**Reply by markuswustenberg** [reply]
No worries, and you're welcome. 🙂
